addons/packages/pinniped/0.22.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: docker.io/getpinniped/pinniped-server:v0.12.1
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v0.12.1
+          url: docker.io/getpinniped/pinniped-server:v0.12.1
+  image: index.docker.io/getpinniped/pinniped-server@sha256:8b4ee3b279d8d1d1f1c65d95f8611a99e00c6d2fbb5dbf974ad76ac4ca563d73
```

Comment:
`ytt` or `imgpkg` or `kbld`, I forget, will update this just fine from upstream container image.

Reply:
`kbld` did it (see package README.md). Good catch!